clients/rust/Cargo.toml

```diff
@@ -16,7 +16,7 @@ serde = ["dep:serde", "dep:serde_with"]
 test-sbf = []
 
 [dependencies]
-anchor-lang = { version = "0.30.0", optional = true }
+anchor-lang = { version = "0.31.1", optional = true }
```

Comment:
Have you built with Anchor and confirmed it builds?

Reply:
yes